gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,5 @@
-#Thu Jun 16 09:15:57 CEST 2016
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-2.14-bin.zip
+distributionUrl=http\://services.gradle.org/distributions/gradle-4.3.1-bin.zip
```

Comment:
better to keep the https here.

Reply:
This is a weird problem with travis where it isn't able to pull it down. I encountered this in the process of open-sourcing commons and was able to resolve it. Can you send me the .travis.yml from there?

Reply:
```

Reply:
Oh, hmmm. I didn't by any chance just change https -> http in that repo, did I? I could have sworn I have to flip a few more things around.